plugins/fileservices/fileservices.cpp

```diff
@@ -3140,3 +3140,56 @@ FILESERVICES_API void FILESERVICES_CALL fsClearExpireDays(ICodeContext * ctx, co
     else
         throw makeStringExceptionV(0, "ClearExpireDays: Could not find logical file %s", lfn.str());
 }
+
+bool getDefaultValue(const char * processName, const char * propertyName, StringBuffer & strDefaultValue)
+{
+    // Returns with true if a default value retrieved and false if not.
+
+    #ifdef _CONTAINERIZED
+        // TBD
```

Comment:
I think the implementation of this should be trivial.
e.g. in fsGetNoCommonDefault():
```
    return getComponentConfigSP()->getPropBool("@noCommon", true);
```

but should be tested.

Reply:
Changed